CHANGES

```diff
@@ -9,6 +9,17 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) X509 certificates signed using SHA1 are no longer allowed at security
+     level 1, the highest security level it's accepted at is 0.
```

Comment:
Just say "at security level 1 and above", and delete the rest of the first sentence.